### ISSUE TITLE
Request for the patch to be able to dump and read the json files from the s06, s07 and s70 folders. 要求补丁能够转储和读取 s06, s07 和 s70 文件夹中的 json 文件

### ISSUE THREAD

**Issue by ToastyBuns3939** [chinosk6/scsp-localify#8]

**Comment by chinosk6** [comment]
Do you know what story/text the json files in these folders about?

**Comment by ToastyBuns3939** [comment]
- s06 is filled with those schedule recommendations your idols give you during produce mode.
- s07 has some generic lines used in produce mode.
- s70 contains the individual characters home screen quips. I know it's already in the localify file, but the files here are already separated by characters so you know who is saying what.

[i've compiled all of the texts i dumped manually here if you want to check them out](
https://docs.google.com/spreadsheets/d/1-9RUhuyvV8UxiJggC0Y_s_hqDpIk2k6ZXCI7Rc2SmIw/edit?usp=sharing)

**Comment by darwintree** [comment]
> * s06 is filled with those schedule recommendations your idols give you during produce mode.
> * s07 has some generic lines used in produce mode.
> * s70 contains the individual characters home screen quips. I know it's already in the localify file, but the files here are already separated by characters so you know who is saying what.
> 
> [i've compiled all of the texts i dumped manually here if you want to check them out](https://docs.google.com/spreadsheets/d/1-9RUhuyvV8UxiJggC0Y_s_hqDpIk2k6ZXCI7Rc2SmIw/edit?usp=sharing)

Hi, this is not related to the issue, but would know currently if it is possible to know the speaker for other folder, for example, s01?

**Comment by ToastyBuns3939** [comment]
`Hi, this is not related to the issue, but would know currently if it is possible to know the speaker for other folder, for example, s01?`
Unfortunately the game .json files didn't list off the characters inside the conversations aside from the side characters that shows up.

**Comment by darwintree** [comment]
> > Hi, this is not related to the issue, but would know currently if it is possible to know the speaker for other folder, for example, s01?
> 
> Unfortunately the game .json files didn't list off the characters inside the conversations aside from the side characters that shows up.

Yeah, I know that. But as we can see the speaker in internal game logs, anyhow there must be some methods that we can link speaker with string id. I've tried but did not find a precise solution, with which enza translation workflow could be used for s4p with a adapter. Anyhow thank you for responding

**Comment by ToastyBuns3939** [comment]
> Yeah, I know that. But as we can see the speaker in internal game logs, anyhow there must be some methods that we can link speaker with string id.

Actually yeah this would be super helpful to have. You should submit a feature request for this.

i've just been mostly grasping in the dark whenever there's a group conversation, so this would save up on a lot of time.

**Comment by chinosk6** [comment]
How did you find these three folders?
I searched for the file through the key of `localify.json`, but did not find the `s06`, `s07`, and `s70` folders.

**Comment by ToastyBuns3939** [comment]
> How did you find these three folders?

I used [this](https://github.com/LukeFZ/PrismCatalogDownloader) to download all of the game files. Including every voice clips and JSON files used by the game. 

Your tool also dumped them to `dumps\autoDumpJson\Scenario` whenever i encounter the commus ingame.

You probably couldn't find them in localify.json because their title names are automated using these.
![image](https://github.com/chinosk6/scsp-localify/assets/93896872/31c3a430-42a7-4e33-b95d-a3bf2ed01d91)

**Comment by chinosk6** [comment]
fixed. binary test files [here](../actions/runs/8506411478)
